ui/src/common/layouts/CollectionsMenu/CollectionsMenu.scss

```diff
@@ -40,20 +40,28 @@
   }
 
   .dropdown-menu-item {
-    padding-top: 0.3em;
-    paddig-bottom: 0.3em;
+    line-height: 2.5rem;
+    margin-top: 0.25rem;
+    padding: 0 1rem;
     &:hover {
       background-color: $layout-header-background !important;
     }
   }
 
+  .ant-dropdown-menu-item > a {
+    display: unset;
```

Comment:
why is that necessary?

Reply:
Because without ```display: unset``` it has ```display: block``` by default and the underline is much lower than the link